src/utils/fen.ts

```diff
@@ -1,4 +1,5 @@
 export const emptyFen = '8/8/8/8/8/8/8/8 w - - 0 1'
+export const initialPositionFen = 'rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1'
```

Comment:
Please use instead the constant already exported in `src/lichess/variant` as `standardFen`